frontend/src/components/widget/InlineTabWrapper.js

```diff
@@ -114,6 +116,16 @@ class InlineTabWrapper extends PureComponent {
             let { validStatus } = deleteResponse.data[0];
             updateDataValidStatus('master', validStatus || { valid: true });
             this.updateTable(true);
+            // calling getData to have the updated included tabs
+            getData({
+              entity: 'window',
+              docType: windowId,
+              docId: docId,
+            }).then(({ data }) => {
+              const response = data[0];
+              response.includedTabsInfo &&
+                updateDataIncludedTabsInfo('master', response.includedTabsInfo);
+            });
```

Comment:
IMHO another API call is not needed. You already have the info returned by DELETE endpoint.

![image](https://user-images.githubusercontent.com/1244701/120475546-29ee2500-c3b2-11eb-8bb0-101391c3ed6b.png)

![image](https://user-images.githubusercontent.com/1244701/120475586-36727d80-c3b2-11eb-9bef-426a5888a004.png)

Reply:
You are right. Just pushed fix . Pls re-check